magento/magento2
# Resolve warnings in PHPMD execution Fixes #33430

### Description (*)


fix the exceptions mentioned in https://github.com/magento/magento2/issues/33430

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#33430

### Manual testing scenarios (*)

1. Install Magento as 2.4-develop
2. run `vendor/bin/phpmd app/code ansi dev/tests/static/testsuite/Magento/Test/Php/_files/phpmd/ruleset.xml`

You should see no execution errors or warnings, just phpmd violations

### Questions or comments


### Contribution checklist (*)
